test/common/upstream/health_check_fuzz.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "test/fuzz/common.proto";
+import "envoy/config/core/v3/health_check.proto";
+
+message Respond {
+  test.fuzz.Headers headers = 1;
+  string status =
+      2 [(validate.rules).string.pattern = "([0-9] | [ ])*"];
```

Comment:
It probably doesn't matter much, but I thought that the codec makes sure that status is a number.

Reply:
https://github.com/nodejs/http-parser/blob/d0f3b0548d41acd1d3992f6c36b192620500be58/http_parser.c#L852 Asra linked me this - seems like empty characters are handled?